third_party/move/move-vm/runtime/src/data_cache.rs

```diff
@@ -69,10 +71,19 @@ impl<'r> TransactionDataCache<'r> {
     ///
     /// Gives all proper guarantees on lifetime of global data as well.
     pub(crate) fn into_effects(self, loader: &Loader) -> PartialVMResult<ChangeSet> {
-        let resource_converter = |value: Value, layout: MoveTypeLayout| -> PartialVMResult<Bytes> {
+        let resource_converter = |value: Value,
```

Comment:
Why not do this in into_custom_effects? I added that function precisely for that reason :)

Reply:
I didn't understand. The into_custom_effects takes a resource_converter as input. The into_effects function first creates a resource_converter and then passes it as input to into_custom_effects.

Reply:
Done.